pandas/api/io/__init__.py

```diff
@@ -0,0 +1,2 @@
+""" Public IO API """
+from pandas.formats.style import Styler  # noqa
```

Comment:
Huh, thought I removed that. Also might have just messed up my rebase. This isn't supposed to be there. It's supposed to be `pandas.io.api`